src/commands/dev/lib/dev-server.ts

```diff
@@ -398,13 +398,15 @@ export default class DevServer {
 
     // Retrieve the path of the native module
     const modulePath = await getModuleForNSFW(this.output);
+    const nowJson = await this.getNowJson();
```

Comment:
I had a bug before when retrieving now.json before `this.env` was assigned because of the validator. Does this fix that bug?

Reply:
Fixing.

Reply:
I personally prefer a more functional approach: the validators should accept all parameters and not rely on `this` state.